lib/src/component/dom_components.dart

```diff
@@ -57,8 +57,6 @@ class SvgProps extends component_base.UiProps with DomPropsMixin, ReactPropsMixi
 
 /// A class that provides namespacing for static DOM component factory methods, much like `React.DOM` in React JS.
 abstract class Dom {
-  /// Returns a new [DomPropsMixin] that renders an `<a>` tag with getters/setters for all DOM-related React props
-  static DomProps a() => new DomProps(react.a);
```

Comment:
This isn't a breaking change since `SvgProps` is a child of `DomProps`, but I suppose we could keep existing factories and just add name spaced ones like you mention.